### DISCUSSION TITLE
Query about the tranistion wavelength's calculated by the ARC package.

### DISCUSSION THREAD

**Discussion by nishant-joshi** [nikolasibalic/ARC-Alkali-Rydberg-Calculator#135] [Q&A]
Dear Nikola,

I am a Phd student at the Raman Research Institute, Bangalore, India. I have a query about the numbers calculated using the Alkali Rydberg calculator and also the Online Atom calculator.

We have a Lithium experiment in our lab.

For the purpose of initial benchmarking I was trying to calculate the 2 S_{1/2}->2 P_{3/2} transition wavelength of Lithium7 and Lithium6. This is the D2 transition.

I know that 2 S_{1/2}->2 P_{3/2} transition wavelengths of Lithium7 and Lithium6 differ by around 10GHz. Please see figure 1 of the following article: https://journals.aps.org/pra/abstract/10.1103/PhysRevA.75.052508

But when I calculate the 2 S_{1/2}->2 P_{3/2} transition wavelengths for Lithium7 and Lithium6 using ARC then I get the same number. I used the following code:

%matplotlib inline
import matplotlib.pyplot as plt
import numpy as np
from IPython.core.display import display, HTML

atom1 = Lithium6()
atom2=Lithium7()
Lithium6 2S_{1/2} -> 2P_{3/2} transition...

print("Li6 Excitation 2S_{1/2} -> 2P_{3/2}")
n1 = 2; l1 = 0; j1 = 0.5; #Initial State
n2 = 2; l2 = 1; j2 = 1.5; #Final State
print("lambda = %.3f nm" % (atom1.getTransitionWavelength(n1,l1,j1,n2,l2,j2)*1e9))
print("omega/2pi = %.3f THz" % (atom1.getTransitionFrequency(n1,l1,j1,n2,l2,j2)*1e-12))
Lithium7 2S_{1/2} -> 2P_{3/2} transition...

print("Li7 Excitation 2S_{1/2} -> 2P_{3/2}")
n1 = 2; l1 = 0; j1 = 0.5; #Initial State
n2 = 2; l2 = 1; j2 = 1.5; #Final State
print("lambda = %.3f nm" % (atom2.getTransitionWavelength(n1,l1,j1,n2,l2,j2)*1e9))
print("omega/2pi = %.3f THz" % (atom2.getTransitionFrequency(n1,l1,j1,n2,l2,j2)*1e-12)

I tried to double-check it using the Online Atom calculator but I get the same result.

The numbers which ARC gives for Lithium7 are fine but for Lithium6 they are off by 10GHz.

This same issue is there in the case of Rubidium 85 and Rubidium 87 as well.

Can you please tell me if I am missing something or whether it is the limitation of the ARC package.

Nishant

Raman Research Institute,
Bangalore,
India.

**Comment by nikolasibalic** [comment]
This is answered now in the[ identical issue](https://github.com/nikolasibalic/ARC-Alkali-Rydberg-Calculator/issues/134). For any follow-up discussion please re-open that issue or add comment in the issue. Closing discussion here.
